kube/services/revproxy/gen3.nginx.conf/arborist-service.conf

```diff
@@ -92,6 +92,25 @@ location = /authz/mapping {
     proxy_pass $upstream/auth/mapping;
 }
 
+location = /authz/policy {
+    if ($csrf_check !~ ^ok-\S.+$) {
+        return 403 "failed csrf check";
+    }
+
```

Comment:
You could limit this to GET explicitly here if that's the expectation.

Reply:
woops, thanks, i'll add that